packages/react-storage/src/components/StorageBrowser/controls/__tests__/PaginationControl.spec.tsx

```diff
@@ -1,20 +1,31 @@
 import React from 'react';
 import { render, screen } from '@testing-library/react';
-import { PaginateControl } from '../Paginate';
-
-import { ControlProvider } from '../../../context/control';
+import { PaginationControl } from '../PaginationControl';
+import { ControlsContextProvider } from '../context';
+import { ControlsContext } from '../types';
 
 describe('PaginationControl', () => {
   it('renders the PaginationControl', async () => {
+    const contextValue: ControlsContext = {
+      data: {
+        pagination: {
+          currentPage: 1,
+          disableNext: false,
+          disablePrevious: false,
+          handlePaginateNext: jest.fn(),
+          handlePaginatePrevious: jest.fn(),
+        },
+      },
+      actionsConfig: { type: 'LIST_LOCATIONS', isCancelable: false },
+    };
+
     render(
-      <ControlProvider actions={{}}>
-        <PaginateControl />
-      </ControlProvider>
+      <ControlsContextProvider {...contextValue}>
+        <PaginationControl />
+      </ControlsContextProvider>
     );
 
-    const nav = screen.getByRole('navigation', {
```

Comment:
Why are we changing the lookup?

Reply:
The component was no longer appearing with the navigation role after changes. Made this change to focus on other failing tests and neglected to go back and fix it. Will fix this

Reply:
This has been fixed / reverted